tests/fixtures/stubs/extending_plugin_entrypoint.php

```diff
@@ -8,7 +8,7 @@
 class ExtendingPluginRegistration implements PluginEntryPointInterface
 {
     /** @return void */
-    public function __invoke(RegistrationInterface $r, SimpleXMLElement $config = null)
+    public function __invoke(RegistrationInterface $r, ?SimpleXMLElement $config = null)
```

Comment:
Well, this one should

Reply:
Why this but not `tests/fixtures/stubs/DomainException.php` ? isn't it the same case ( changing the argument typing using the default value )?

Reply:
The signature here is dictated by `PluginEntryPointInterface` that got changed.